### DISCUSSION TITLE
Cargar rango de fechas

### DISCUSSION THREAD

**Discussion by qarlos77** [tmijail/Odoo-Power-BI-Connector#16] [Q&A]
Hola! con qué función podría hacer un llamado a la base de datos para que solo me traiga determinado rango de fechas. Al tener más de 200k entradas, esto causa que el Odoo se cuelgue. El modelo que estoy utilizando es pos.payment
Gracias!

**Comment by tmijail** [comment]
Le podes pasar un conjunto de filtros ([search domain](https://www.odoo.com/documentation/16.0/developer/reference/backend/orm.html#reference-orm-domains) en terminología de Odoo) a `search_read`.

Algo así:
```
= search_read(
    "pos.payment",
    {
        {"payment_date" , ">=", "2023-01-01"},
        {"payment_date" , "<=", "2023-12-31"}
    }
)
```
